icu4c/source/i18n/number_longnames.cpp

```diff
@@ -54,6 +54,14 @@ static int32_t getIndex(const char* pluralKeyword, UErrorCode& status) {
     }
 }
 
+double absDouble(double number) {
```

Comment:
Quick single comment: I saw there's a uprv_fabs declared in putilimp.h - I think rather use that than creating a new abs-of-double function.

Reply:
done